com.civicrm.hrjobroles/CRM/Hrjobroles/DAO/HrJobRoles.php

```diff
@@ -266,7 +266,7 @@ static function &fields()
   {
     if (!(self::$_fields)) {
       self::$_fields = array(
-        'hrjobroles_id' => array(
+        'hr_job_roles_id' => array(
```

Comment:
DAO files should be generated from the XML files and they should not be edited manually

Reply:
Yes that is correct, but this extension have not followed that process for a while now. Plans are in place to ensure its properly migrated to follow appropriate standards. Until then, we just carefully ensure implementations are well tested.